### PR TITLE
Convert Rnw vignette to Rmd and bump version

### PR DESCRIPTION
**Pull request by jwokaty**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-Version: 1.41.0
+Version: 1.41.1
 Author: Martin Morgan
 Maintainer: Bioconductor Package Maintainer <maintainer@bioconductor.org>
 Description: This package uses the source code of zlib-1.2.5 to create
   libraries for systems that do not have these available via other
   means (most Linux and Mac users should have system-level access to
   zlib, and no direct need for this package). See the vignette for
   instructions on use.
+Imports: BiocStyle
```
Comment (LiNk-NY) [inline DESCRIPTION]: This should be `Suggests`.

File: vignettes/UsingZlibbioc.Rmd [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+library is always built on Windows, but on other platforms it is only built
+when provided with the configure option `{r eval=FALSE}--with-libzbioc`, e.g.,
+as
+
+```{r eval=FALSE}
+R CMD INSTALL --configure-args="--with-libzbioc" zlibioc_<...>.tar.gz
+```
+or
+```{r eval=FALSE} 
+> install.packages("zlibbioc_<...>.tar.gz",
```
Comment (LiNk-NY) [inline vignettes/UsingZlibbioc.Rmd]: You can omit the `>` here

**Comment by jwokaty** [comment]
@LiNk-NY Could I ask you to look at my commit to change zlibbioc's Rnw to Rmd? I did everything manually and compared the output of the existing vignette with the output of the one I created.

File: vignettes/UsingZlibbioc.Rmd [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+about portability in 'Writing **R** Extensions') with
+
+```{bash eval=FALSE}
+PKG_CFLAGS+=$(shell echo 'zlibbioc::pkgconfig("PKG_CFLAGS")'|\
+    "${R_HOME}/bin/R" --vanilla --slave)
+PKG_LIBS+=$(shell echo 'zlibbioc::pkgconfig("PKG_LIBS_shared")' |\
+    "${R_HOME}/bin/R" --vanilla --slave)
+```
+
+The *Rsamtools* package is a more complex example illustrating this approach.
```
Comment (LiNk-NY) [inline vignettes/UsingZlibbioc.Rmd]: Perhaps add a link to the package via `r Biocpkg('Rsamtools')`

**Review by LiNk-NY — CHANGES_REQUESTED** [review]
It looks good. Perhaps format the bash code with triple backticks ``` and omit the extra stuff i.e., `{bash ...}`

**Review by LiNk-NY — APPROVED** [review]

**Pull request merged**